### DISCUSSION TITLE
Using swup with React or Next

### DISCUSSION THREAD

**Discussion by kakonbarman** [swup/swup#630] [General]
**Describe the problem**

**Describe the proposed solution**

**Alternatives considered**

**How important is this feature to you?**

- [ ] Nice to have
- [ ] Would make my life a lot easier
- [ ] I cannot use swup without it

**Comment by hirasso** [comment]
Could you be more specific?

**Comment by daun** [comment]
Swup was specifically created to animate page loads on server-rendered multi-page applications. It's not a good fit for frameworks that implement their own client-side routing, like React and Next do, as both routers would just get in each other's way.

**Comment by iamnishantgaharwar** [comment]
Can we use swup for changing background color but their is no page 

Example website: https://brewdistrict24.com/en

**Comment by hirasso** [comment]
@iamnishantgaharwar please create a new discussion if you're having a question.

**Reply by iamnishantgaharwar** [reply]
Okay thanks
